theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsOpacityTokens.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme.tokens.semantic
+
+import androidx.compose.runtime.Stable
+import com.orange.ouds.tokens.OpacityRawTokens
+
+class OudsOpacityTokens(
+    var transparent: Float = OpacityRawTokens.opacity0,
+    var weaker: Float = OpacityRawTokens.opacity100,
+    var weak: Float = OpacityRawTokens.opacity300,
+    var medium: Float = OpacityRawTokens.opacity500,
+    var emphasis: Float = OpacityRawTokens.opacity700,
+    var opaque: Float = OpacityRawTokens.opacity900
+)
+
+enum class OudsOpacityToken {
+    Transparent,
+    Weaker,
+    Weak,
+    Medium,
+    Emphasis,
+    Opaque
+}
+
+@Stable
```

Comment:
I'm not familiar with this annotation. Do you remember why did you add it to this method? (I did not see that it was already the case with the `fromToken` method of `OudsColors`).